src/pages/extensions/iter8/Iter8ExperimentDetails/ExperimentCreatePage.scss

```diff
@@ -0,0 +1,7 @@
+.pf-c-switch__toggle {
```

Comment:
This is the style that turns orange the default styles, I don't think is needed.